randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -297,7 +300,13 @@ Extra - room_id: [9]
       All of the following:
           # TODO:  damage reqs
           Varia Suit
-          Wall Jump (Beginner) or Can Climb High Jump Wall or Can Use Any Bombs
+          Any of the following:
+              Wall Jump (Beginner) or Can Climb High Jump Wall
+              # Climb Hidden Ladder
+              Knowledge (Beginner) and Can Use Any Bombs
```

Comment:
can you trigger the ladder with bombs without mid air morph/spring?

Reply:
Is this a no or a yes?

Reply:
![image](https://github.com/user-attachments/assets/489d809f-3ba2-4e9b-98f8-ef0de1c246c5)
technically yes, you could just roll off from here and just climb up again.

Reply:
Can you either add that or make a note in the thread?